.gitignore

```diff
@@ -16,3 +16,4 @@
 configure.wrf*
 *.backup
 *.f90
+build.log
```

Comment:
Would you revert this file please. There are a large number of dummy file names that could be used for the build log.

Reply:
Will do.